code/web/src/modules/user/Profile.js

```diff
@@ -31,16 +35,42 @@ const Profile = (props) => (
     </Grid>
 
     <Grid>
+      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
+        {/*<Tile style={{}} image={icon}></Tile>*/}
+        <ImageTile width={300} height={530} shadow={level1} image={`${ APP_URL }/images/stock/men/1.jpg`} />
+      </GridCell>
       <GridCell style={{ padding: '2em', textAlign: 'center' }}>
         <H4 style={{ marginBottom: '0.5em' }}>{props.user.details.name}</H4>
-
+        <p style={{ marginBottom: '0.5em' }}>Personal description goes here</p>
+        {/*Description text should be dynamic from user details ie props.user.details.description */}
+      </GridCell>
+      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
         <p style={{ color: grey2, marginBottom: '2em' }}>{props.user.details.email}</p>
-
-        <Link to={userRoutes.subscriptions.path}>
-          <Button theme="primary">Subscriptions</Button>
-        </Link>
-
+        <p style={{ color: grey2, marginBottom: '2em' }}>123 West 5th St City, State 80654</p>
+        {/*Address text should be dynamic from user details ie props.user.details.shipping */}
+      </GridCell>
+    </Grid>
+    <Grid>
+      {
+        this.props.orderHistory.isLoading
+          ? <Loading/>
+          : this.props.orderHistory.list.length > 0
+              ? this.props.orderHistory.list.map(product => (
+                  <GridCell key={product.id} style={{ margin: '2em', float: 'left' }}>
+                    <H3>{product.name}</H3>
+                    <p>{product.description}</p>
+                    {/* Items will also have a boolean value to indicate if item was kept */}
+                    {/* Render image with opacity based on boolean value */}
+                  </GridCell>
+                ))
+              : <EmptyMessage message="You have not received any crate orders yet." />
+      }
+    </Grid>
+    <Grid>
+      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
+        <Button theme="primary" onClick={props.loadHistory}>Order History</Button>
         <Button theme="secondary" onClick={props.logout} style={{ marginLeft: '1em' }}>Logout</Button>
```

Comment:
It might be a good idea to move the logout button to the header. It seems counter intuitive that the user must navigate to their profile page in order to logout. What do ya'll think?

Reply:
I think that makes a lot of sense, or to at least have it in both locations.